### DISCUSSION TITLE
Auto Tilling manager

### DISCUSSION THREAD

**Discussion by thalesmaoa** [elementary/triage#471] [Q&A]
Hi there!
eOS has the best design ever! However, from time to time I try other distros due to some annoying bugs.
During my trials, I've tested some Auto Tilling manager.

Please, implement a auto tilling manager for eOS. 
It makes tremendous difference for the workflow having it.

I've been using popOS and I will keep it for a while.
Is it possible to implement something?

**Comment by lenemter** [comment]
Duplicate of https://github.com/elementary/gala/issues/316
